Module not found 에러를 해결한다.

## 📑 작업 사항

> 작업할 내용을 작성해주세요

Module not found 에러를 해결한다.

> 에러 내용을 설명해주세요

`Module not found: Error: Can't resolve './styles/index.css' in '/Users/wusi/Desktop/gitkraken/2023-yozm-cafe/client/src' `

## 🎸 기타 사항

> 기타 사항을 작성해주세요

--- a/client/src/index.tsx
+++ b/client/src/index.tsx
@@ -1,6 +1,5 @@
 import ReactDOM from 'react-dom/client';
 import App from './App';
-import './styles/index.css';
 import { worker } from './mocks/worker';
 
 worker.start();

